### DISCUSSION TITLE
Embedding requests within a language (Python, JavaScript, R)

### DISCUSSION THREAD

**Discussion by MarkAPitt** [ngrams-dev/general#4] [Q&A]
A helpful [resource ](https://curlconverter.com/python/)for converting curl commands in the example code to the language of your choice:

**Comment by mtrenkmann** [comment]
Thanks @MarkAPitt and welcome to NGRAMS. My plan is to show example code in other languages when the new documentation is ready. This tool looks helpful. The current GitHub Wiki / Markdown unfortunately has no support for a tabbed pane...

Created issue #5 to keep track.
